app/assets/javascripts/discourse/app/controllers/preferences/profile.js

```diff
@@ -43,30 +44,15 @@ export default Controller.extend({
     }
   },
 
-  @discourseComputed("model.can_change_bio")
-  canChangeBio(canChangeBio) {
-    return canChangeBio;
-  },
+  canChangeBio: alias("model.can_change_bio"),
```

Comment:
Sorry to request another change here, but could you use the `readonly` helper? It's better than alias if we know it's only a one way relationship.

Reply:
said changes done @eviltrout